app/views/layouts/sites/_developer_portals.html.erb

```diff
@@ -12,5 +12,8 @@
 </ul>
 
 <div id="tab-content">
+  <% if content_for :title %>
+    <h1><%= content_for :title %></h1>
```

Comment:
DRY?

Reply:
@guicassolato we need the title inside tab content div

Reply:
you mean extracting this into it's own partial?

Reply:
I mean `if content_for :title` and then `= content_for :title`. In fact, I see this pattern i two other places in the same PR. Perhaps having a partial for that.

Also, I don't know what `content_for` returns, but... could it be truthy for `title == ""`?

Reply:
> Also, I don't know what content_for returns, but... could it be truthy for title == ""?

By default it is nil, if it is an empty String is because someone has set it to that somewhere in the code, but then I guess it would be truthy.

This reminds me to when I read the book `Ruby on Rails Tutorial` by `Michael Hartl` because there was a very similar situation 😝But I read it many years ago so I didn't remember the details. It is in [Chapter 4](https://www.railstutorial.org/book/rails_flavored_ruby). What he does is never ever having an empty title, always having one, even if it is just a default, and it does a helper to don't repeat the same logic everywhere. So the helper looks like:
```ruby
module ApplicationHelper

  # Returns the full title on a per-page basis.
  def full_title(page_title = '')
    base_title = "Ruby on Rails Tutorial Sample App"
    if page_title.empty?
      base_title
    else
      page_title + " | " + base_title
    end
  end
end
```
and then from the view:
```
<title><%= full_title(yield(:title)) %></title>
```
I think we can adapt it to out situation with something like:
```ruby
module ApplicationHelper
  def full_title(page_title = '')
    base_title = 'General'
    title = page_title.empty? ? base_title : (page_title + " | " + base_title)
    "<h1>#{title}</h1>".html_safe
  end
end
```
And in the view: `full_title(yield(:title))` or `full_title(content_for(:title))`
I dunno if the html_safe should be in the helper or the view, but you get the hint 😄

Reply:
@damianpm For me being a partial or a helper I don't really mind much in this case 😄 but we still have the issue of covering the case in which no title is set to be able to have one anyway 😄 and checking for the empty String for example. So it could be something like this for example:
```
- title = content_for(:title).presence || 'General' # This is an example... I don't know what is the general title :)
h1 = title
```

Reply:
No, we don't want a default title when no title is set. we only want that for the page title in the head, which we have. No defaults in the actually rendered title in the main content.